CHANGES.rst

```diff
@@ -55,6 +55,11 @@ https://github.com/zopefoundation/Zope/blob/4.x/CHANGES.rst
 
 - Improve documentation for Zope's error logging services.
 
+- Add ``Paste`` as ``extras_require`` dependency to pull in ``Paste`` when 
+  installing with `pip` and `constraints.txt` to prevent startup errors.
+  (`#734 <https://github.com/zopefoundation/Zope/issues/734>`_)
```

Comment:
Maybe we should mention here that this requires a change in how to install Zope with `pip`

Reply:
Just added a note in there.